src/Cake.AndroidAppManifest/AndroidAppManifest.cs

```diff
@@ -200,7 +200,7 @@ public void WriteToFile(string fileName)
                 IndentChars = "\t",
                 NewLineChars = "\n"
             };
-            using (var stream = File.OpenWrite(fileName))
+            using (var stream = File.Open(fileName, FileMode.Create))
```

Comment:
@lprichar Can you change the `File.Open(string path, System.IO.FileMode mode)` to `File.Open(string path, System.IO.FileMode mode, System.IO.FileAccess access)` so we can specify `FileAccess` as `ReadWrite `? 
See documentation [here](https://docs.microsoft.com/en-us/dotnet/api/system.io.fileaccess?view=netframework-4.7.1) fore more detail.